src/utilities/sequence-editor/extension-points.ts

```diff
@@ -81,14 +81,15 @@ export function inputLinter(
   channelDictionary: ChannelDictionary | null = null,
   commandDictionary: CommandDictionary | null = null,
   parameterDictionaries: ParameterDictionary[] = [],
+  LibrarySequences: LibrarySequence[] = [],
```

Comment:
Casing is incorrect for this variable, should be camelCase.

Reply:
Whoops, I fixed that